src/Symfony/Component/DependencyInjection/CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ CHANGELOG
  * removed support for setting and accessing private services in `Container`
  * removed support for setting pre-defined services in `Container`
  * removed support for case insensitivity of parameter names
+ * removed `AutowireExceptionPass` and `AutowirePass::getAutowiringExceptions()`, use `Definition::addError()` and the `DefinitionErrorExceptionPass` instead
```

Comment:
Not having anything in the 3.x changelog about the deprecation looks weird to me. The changelog should be updated (in a separate branch)

Reply:
See #24399 :)